AdvancedApplications/JDBCApplication/runExampleJDBCApp

```diff
@@ -1,6 +1,6 @@
 echo "Compile java code"
 if [ -z ${ESLIB} ]; then
-ESLIB=${SPARK_HOME}/jars/ibm-db2-eventstore-client-spark-2.2.1-2.0.0.jar
+ESLIB=${SPARK_HOME}/jars/ibm-event_2.11-1.0.jar
```

Comment:
why rename it ? Can we change the wget to download and rename it at the same time ? 

Additionally, it would be nice to be able to provide a specific version of the JAR to download when the container is started, but that can be done in a different PR.

Reply:
OK. Undid the renaming of the client spark jar file.
The file should be updated in maven.